packages/push-notifications/package.json

```
@@ -26,15 +26,18 @@
     "test": "jest"
   },
   "devDependencies": {
-    "@aries-framework/core": "^0.2.0",
-    "@aries-framework/node": "^0.2.0",
+    "@aries-framework/core": "^0.4.0",
+    "@aries-framework/indy-sdk": "^0.4.0",
+    "@aries-framework/node": "^0.4.0",
     "@types/jest": "^27.0.2",
+    "@types/node-fetch": "^2.6.3",
+    "indy-sdk": "^1.16.0-dev-1655",
     "ts-node-dev": "^1.1.8",
     "tslog": "^3.2.2",
     "typescript": "~4.4.2"
   },
   "peerDependencies": {
-    "@aries-framework/core": "^0.2.0"
+    "@aries-framework/core": "0.4.0"
```

```suggestion
    "@aries-framework/core": "^0.4.0"
```

or is there an unknown quirk I am not familiar with of peerDeps with `^`?